packages/ckeditor5-table/src/tablelayout/tablelayoutui.ts

```diff
@@ -36,16 +50,20 @@ export default class TableLayoutUI extends Plugin {
 		return true;
 	}
 
+	/**
+	 * @inheritDoc
+	 */
+	public static get requires() {
+		return [ TablePropertiesUI ] as const;
```

Comment:
I'm not sure if we should enforce the table properties feature. Maybe we should integrate with it only if it is loaded, otherwise register a separate dropdown?

Reply:
Added fallback:

https://github.com/user-attachments/assets/813dc931-1caa-4815-b01e-1330cdbaeabd